docker-compose.yaml

```diff
@@ -2,35 +2,26 @@ version: "3.8"
 services:
   d4:
     image: dillonad/d4bot:latest
+    build:
+      context: .
+      dockerfile: Dockerfile
     container_name: d4
-    env_file:
-      - .env
-      - .env.local
-    networks:
-      - d4bot
-
-  otel-collector:
-    container_name: otel-collector
-    image: ghcr.io/open-telemetry/opentelemetry-collector-releases/opentelemetry-collector-contrib:latest
-    volumes:
-      - ./otel-config.yaml:/etc/otelcol-contrib/config.yaml:Z
-    ports:
-      - "4317:4317"
+    environment:
+      OTEL_ENDPOINT: jaeger-ui:4317
+      OTEL_EXPORTER_INSECURE: "true"
+      BOT_TOKEN: ${BOT_TOKEN}
     networks:
       - d4bot
+    depends_on:
+      - jaeger-ui
 
   jaeger-ui:
```

Comment:
I'm not sure why I set this up with Jaeger. How about we use Honeycomb? If we send directly to Honeycomb, we can get ride of `docker-compose` altogether?

Reply:
Pulled out everything we don't need if we're honeycomb-ing.